lib/ActivityStream.jsm

```diff
@@ -313,13 +313,6 @@ const PREFS_CONFIG = new Map([
       value: "https://incoming.telemetry.mozilla.org/submit",
     },
   ],
-  [
-    "telemetry.ping.endpoint",
```

Comment:
Curious, does the browser clean up this pref if you delete it here like you are?

Or does it stick around in the user's profile?

Reply:
I think the browser will delete it if this pref no longer exists.

Reply:
Good to know!